modules/ROOT/pages/organization-in-bonita-bpm-applications-overview.adoc

```diff
@@ -1,7 +1,8 @@
 = Organization Management
+:page-aliases: organization-in-bonita-bpm-portal-overview.adoc
 :description: In Bonita Portal as in Bonita Administrator Application, a user with the _Administrator_ profile can view all the entities of the deployed organization.
 
-In Bonita Portal as in Bonita Administrator Application, a user with the _Administrator_ profile can view all the entities of the deployed organization.
```

Comment:
We can keep "in Bonita Administrator Application, a user with the _Administrator_ profile can view all the entities of the deployed organization." and only remove "In Bonita Portal as "

Reply:
Will be done in another pr